scipy/optimize/_lsq/lsq_linear.py

```diff
@@ -120,8 +127,9 @@ def lsq_linear(A, b, bounds=(-np.inf, np.inf), method='trf', tol=1e-10,
             * -1 : a lower bound is active.
             *  1 : an upper bound is active.
 
-        Somewhat arbitrary because it is determined within a tolerance
-        threshold.
+        Might be somewhat arbitrary for `trf` method as it generates a sequence
```

Comment:
Yeah, OK, probably it is covered by the rule like "the Pacific Ocean" (searched for it now). I really didn't know how to write it correctly.